src/organisms/tables/ComparisonTable/comparison_table.module.scss

```diff
@@ -0,0 +1,63 @@
+@import 'src/assets/stylesheets/global';
+
+.comparison-table {
+  border: 1px solid $neutral-5;
+  border-radius: 6px;
+  padding: 0;
+
+  .child-wrapper:last-child {
+    .row {
+      border-bottom: none;
+    }
+  }
+}
+
+.header {
+  padding: 24px 30px;
+  border-right: 1px solid $neutral-5;
+  border-bottom: 1px solid $neutral-5;
+
+  &:last-child {
+    border-right: none;
+  }
+}
+
+.row {
+  padding: 18px 30px;
+  position: relative;
+  border-right: 1px solid $neutral-5;
+  border-bottom: 1px solid $neutral-5;
+
+  &:last-child {
+    border-right: none;
+  }
+}
+
+.layout {
+  padding: 0;
+}
+
+@media #{$small-only} {
+  .comparison-table {
+    border: none;
+  }
+
+  .header {
+    border: none;
+
+    &:not( :first-child ) {
+      display: none;
```

Comment:
@drewdrewthis ahh yeah this isn't ideal but the reason I did it is because in the header, I want  everything to be `display:none` on small screens except the `Why apply with us`. I didn't want to complicate the `ComparisonTableRow` component by trying to check the children & add a different class name based on the index or something- maybe there's a better way?